src/persist/src/file.rs

```diff
@@ -204,9 +209,19 @@ impl FileBuffer {
     /// that another FileBuffer is already using the dir), an error is returned
     /// from `new`.
     ///
-    /// The contents are `lock_info` are stored in the LOCK file and should
+    /// The contents of `lock_info` are stored in the LOCK file and should
     /// include anything that would help debug an unexpected LOCK file, such as
     /// version, ip, worker number, etc.
+    ///
+    /// The data is stored in a separate buffer file, and is formatted as:
```

Comment:
This is still missing a piece around there being a series of (concatenated?/sequential?) entries, each of which is structured as what you have below.